setup/prepare-crc.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#This script configures CRC for running app-studio in it
+#It cofigures the CRC to have better memory and CPUs 
+#It also configures CRC to have required addons.
+
+#TODO: Check for CRC installation and binary location and set it to CRC-BINARY
+CRCBINARY=$(readlink -f ~/.crc/bin/crc)
+
+echo $CRCBINARY
+#Get CRC VMs configured in local machine
+$CRCBINARY setup
+#Set memory and CPU allowance for CRC
+$CRCBINARY config set memory 16384
```

Comment:
These are recommended minimums.  I'm not sure what the actual minimums are.
You can certainly go higher.